website/package.json

```diff
@@ -34,25 +33,25 @@
 		"@types/node": "^18.18.0",
 		"@types/react": "^18.2.23",
 		"@types/react-dom": "^18.2.8",
-		"@uiw/react-codemirror": "4.20.2",
+		"@uiw/react-codemirror": "4.21.21",
 		"@vitejs/plugin-react": "^4.2.1",
 		"autoprefixer": "^10.4.16",
 		"codemirror-lang-rome-ast": "0.0.6",
 		"fast-diff": "^1.3.0",
-		"hast-util-to-html": "^8.0.4",
-		"hastscript": "^8.0.0",
+		"hast-util-to-html": "^9.0.0",
+		"hast-util-to-string": "^3.0.0",
+		"hastscript": "^9.0.0",
 		"html-escaper": "^3.0.3",
 		"lang-rome-formatter-ir": "0.0.2",
-		"mdast-util-to-hast": "^12.3.0",
+		"mdast-util-to-hast": "^13.1.0",
 		"mermaid": "^9.4.3",
 		"postcss": "^8.4.30",
 		"prettier": "^3.1.0",
-		"react": "^17.0.2",
-		"react-dom": "^17.0.2",
-		"rehype-autolink-headings": "^6.1.1",
-		"rehype-slug": "^5.1.0",
-		"rehype-toc": "^3.0.2",
-		"remark-toc": "^8.0.1",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
```

Comment:
Seems like the playground doesn't work anymore. Unless you want to spend time understanding the issue, maybe we can revert the change

Reply:
The issue doesn't seem related to updating the React dependencies, reverting them doesn't fix, nor does loading the Playground without `<StarlightPage>` - my best guess it might be Vite-related? But hard to know without context on how the Playground was built.